third_party/move/move-compiler-v2/src/function_checker.rs

```diff
@@ -391,3 +424,23 @@ fn not_a_friend_error(
     );
     cannot_call_error(env, &why, sites, caller, callee);
 }
+
+fn diff_package_error(
+    env: &GlobalEnv,
+    sites: &BTreeSet<NodeId>,
+    caller: &FunctionEnv,
+    callee: &FunctionEnv,
+) {
+    let why = "they are from different packages";
+    cannot_call_error(env, why, sites, caller, callee);
+}
+
+fn diff_addr_error(
```

Comment:
Why is this still an error?

Reply:
This is used when calling `public(package)` function from a different address.